test/test-lib/fixtures.ts

```diff
@@ -113,6 +148,64 @@ const loaders: Dictionary<LoaderFunc> = {
 	},
 };
 
+loaders['devices'] = async (jsonData, fixtures) => {
```

Comment:
Can you move this within the object literal creation like the other loaders please

Reply:
moved.